.github/workflows/python-ci.yml

```diff
@@ -17,7 +17,7 @@ jobs:
       - name: Setup Python
         uses: actions/setup-python@v3
         with:
-          python-version: '3.9.x'
+          python-version: '3.11.x'
```

Comment:
This needs to stay on 3.9: people that migrated their installation in place from the original BirdNET-Pi are still on 3.9.

Reply:
3.9 is about 4 years old. When can we move to a more up to date version? Should updating Python and/or other dependencies be part of the update script.

Reply:
What Nachtzuster means (I think) is that birdnet pi has an incremental auto update feature. So if we change the dependency to a newer version of python incremental update won't work and it will break functionality for older users

Reply:
I just think at some point we're going to want to have an update, if not for better Python features than to avoid any potential security issues. It doesn't need to be bleeding edge but generally speaking staying back leveled isn't ideal.